src/components/Journal/JournalEntryForm/journalEntryForm.scss

```diff
@@ -0,0 +1,35 @@
+.Layer__JournalEntryForm {
+  padding-top: var(--spacing-xl);
+
+  .Layer__JournalEntryForm__Row {
+    padding: 0 var(--spacing-xl);
+    display: flex;
+    gap: var(--spacing-md);
+
+    > * {
+      flex: 0 0 clamp(16rem, 100%, 24rem);
+    }
+  }
+
+  @container (max-width: 760px) {
+    .Layer__JournalEntryForm__Row {
+      flex-direction: column;
+
+      > * {
+        flex: 1;
+        max-width: clamp(12rem, 100%, 16rem);
+      }
+    }
```

Comment:
@sarahraines would love your take: I've made this form responsive (ugly, but usable) with this pattern of basically switching from a flex-row to a flex-column if the screen gets too narrow and applying it in a couple places.

Downside is that you can't use `<HStack>` or `<VStack>` because it's effectively toggling between them. I'm open to either

1. Making this a reusable pattern
2. Not using this pattern at all
3. Keeping this as a custom styled div